kubernetes-kit-starter/src/main/java/com/vaadin/kubernetes/starter/sessiontracker/SessionSerializationCallback.java

```diff
@@ -0,0 +1,49 @@
+package com.vaadin.kubernetes.starter.sessiontracker;
+
+/**
+ * Callbacks that are called after a successful serialization and
+ * deserialization or when an error happens during the serialization or
+ * deserialization. Each callback has a default method implementation. A bean
+ * needs to be crated from a class implementing this interface.
+ */
+public interface SessionSerializationCallback {
+
+    /**
+     * The default implementation of this interface, which is used when there is
+     * no bean provided from other implementation.
```

Comment:
What about adding a note that this is a no-op implementation? Just to make it evident that by default, nothing is done.
```suggestion
     * The default no-op implementation of this interface, which is used when there is
     * no bean provided from other implementation. It does not perfom any operation.
```

Reply:
Updated the javadoc.